packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -47,6 +47,7 @@ import {
     DataCorruptionError,
     extractSafePropertiesFromMessage,
     DataProcessingError,
+    UsageError,
 } from "@fluidframework/container-utils";
```

Comment:
You would probably want to start with only containerRuntime.ts changes (i.e. remove deltaMnager.ts changes) as it's much harder to make them optional.
The way containerRuntime.ts is implemented - it gives a way for people to disable it, at least temporarily and fix their issues outside the process of picking latest version of FF